### ISSUE TITLE
Gambit does not start on Windows XP

### ISSUE THREAD

**Issue by jgesswein** [feeley/gambit#119]
Gambit (v4.7.5, Microsoft Windows version) cannot start on Windows XP. The system indicates that Gambit (both gsi and gsc) is not a valid Windows application. This is caused by the Windows subsystem found in the executable. The subsystem required by Gambit is is “6.0” but Windows XP’s subsystem is only “5.1”. Support for Windows XP is built into SP3 for Visual C++ 2012 Express.

The version created by MinGW launches and works just fine on Windows XP. The Windows subsystem is set to “4.0” for this version.

This behaviour is observed also for version 4.7.3 and 4.7.4. I did not check more versions but probably all version created using Visual C++ 2012 Express have this problem.

**Comment by m-i-k-a-e-l** [comment]
jgesswein, which Windows version a Visual C++-produced binary requires, is specified by command line arguments to it. (Or alternatively maybe via a DEFINE in Gambit.h etc.).

You have the build script in misc/*.bat.

Would you want to look into this and come up with a patch?

**Comment by jgesswein** [comment]
Version 4.7.6 cannot be started on Windows XP as well. I’ll have a look at the build script.

Which script is used to create release binaries and what is the studio version? I’ll use vcexpress.bat and VC++ 2010/2013 Express.

**Comment by feeley** [comment]
Release binaries for windows are created by the script prebuilt/windows/build .  It needs to be executed from a mingw shell.  You will have to adjust these definitions in the script:

```
PFDIR="/Program Files (x86)"

VCDIR="$PFDIR/Microsoft Visual Studio 12.0/VC"
WKDIR="$PFDIR/Windows Kits/8.1"

MSYS="/msys"

ADDPATHMINGW32="/c/mingw32/bin:"
ADDPATHMINGW64="/c/mingw64/bin:"

ADDPATHVC32="/c$VCDIR/bin:"
ADDPATHVC64="/c$VCDIR/bin/x86_amd64:"

COMMONPATH="/c$MSYS/bin:"
```

Please let me know what has to be changed to get it working on Windows XP.

**Comment by feeley** [comment]
Fixed by b496f408a336f2f59dac50489072b21d2456e2bf.
